packages/pds/src/pipethrough.ts

```diff
@@ -23,12 +23,23 @@ import {
 import AppContext from './context'
 import { ids } from './lexicon/lexicons'
 import { httpLogger } from './logger'
+import { AuthScope } from './auth-verifier'
+
+const getStandardAccess = (ctx: AppContext, path: string) => {
```

Comment:
I think what I'd prefer to do here is just add a route for handling `createReport` to the PDS with the correct auth scope on it. This will keep it from falling back to this generic proxy logic.

Two benefits here:
- we keep this route-specific complexity out of the proxy code & can keep it more generic
- we avoid running the `getStandardAccess` fn on every proxy request which will keep them a bit lighter

We can still use the `pipethrough` function from this file to help with actually proxying the request - you can find examples in `getFeed` and `getRecord`